lockfile/src/lib.rs

```diff
@@ -38,6 +40,8 @@ pub enum LockfileFormat {
     #[serde(rename = "nuget")]
     #[serde(alias = "msbuild")]
     Msbuild,
+    #[serde(rename = "gosum")]
+    GoSum,
```

Comment:
🤔 Maybe it would be better to just change the `rename_all` on this enum to use `lowercase` instead of `camelCase`?

All of the other variants are one "word" (or at least they only have one capital letter), so this is the first variant where the distinction between `camelCase` and `lowercase` actually matters. If we want this one to be all lowercase, we might as well make that the decision going forward.

Reply:
good call, done